programmable_video/lib/src/camera_capturer.dart

```diff
@@ -89,10 +89,9 @@ class CameraCapturer implements VideoCapturer {
   }
 
   /// Takes a photo from the camera capturer.
-  Future<dynamic> takePhoto(int imageCompression) async {
```

Comment:
Why remove this?

Reply:
It was harder to pass through the imageCompression option to the new Android implementation. Also, we're not really use of this option in the first place, as it was defaulted to 80 before.